src/core/featuremodel.h

```diff
@@ -289,6 +293,11 @@ class FeatureModel : public QAbstractListModel
     bool startEditing();
     void setLinkedFeatureValues();
     void updateDefaultValues();
+    void updateGeometryLocked();
+
+    bool mGeometryLocked = false;
+    bool mGeometryLockedByDefault = false;
+    QString mGeometryLockedExpression;
```

Comment:
```suggestion
    bool mGeometryLocked = false;
    bool mGeometryLockedByDefault = false;
```

Small docs about these two would help future us to understand what is used when without following the logic.